.kitchen.yml

```diff
@@ -25,6 +28,23 @@ suites:
                 port: 6379,
               }
             ]
+  - name: redis-package
```

Comment:
Well, I'm not very comfortable with this for 2 reasons:
1. I never did it yet :-p
2. I think it's not necessary for a such basic thing